### PR TITLE
Create prozt.json

### PR DESCRIPTION
**Pull request by pxldevv**
Add prozt domain

### CODE REVIEW

**Review by prabincankod — CHANGES_REQUESTED** [review]
CNAME record cannot be a link with directory, it must me plain. or you can use 'URL' key instead of 'CNAME' to redirect your subdomain to the destination of yours.

File: domains/prozt.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "owner": {
+    "username": "github-username",
+    "email": "prozt709@gmail.com",
+  },
+  "record": {
+    "CNAME": "https://www.proz.tk/prozt709/"
```
Comment (prabincankod) [inline domains/prozt.json:7]: either change the 'CNAME' key to 'URL' or make the 'CNAME' key value valid of some sort.

**Review by pxldevv — COMMENTED** [review]
I Changed it

**Comment by Techpro709** [comment]
https://github.com/is-a-dev/register/pull/1600/files/a8a2a7a9ffe151dabaaf97baf3ffeff3e3bfcda6#diff-9d1213b2617ef9d99060005f5fd9195760a9de1e5a5fe725651a8c67361b9631.patch

**Comment by Techpro709** [comment]
https://github.com/is-a-dev/register/pull/1600/files/a8a2a7a9ffe151dabaaf97baf3ffeff3e3bfcda6#diff-9d1213b2617ef9d99060005f5fd9195760a9de1e5a5fe725651a8c67361b9631.diff

**Comment by Techpro709** [comment]
https://patch-diff.githubusercontent.com/raw/is-a-dev/register/pull/1600.patch

**Comment by Techpro709** [comment]
https://patch-diff.githubusercontent.com/raw/is-a-dev/register/pull/1600.patch

**Review by phenax — APPROVED** [review]

**Pull request merged**